<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temperature_control</class>
 <widget class="QWidget" name="temperature_control">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="send_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="send_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="send_address_label">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="send_address_spinbox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="function_type_label">
          <property name="text">
           <string>Функция</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="function_type_combo"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="send_value_label">
          <property name="text">
           <string>Значение(я)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="values_field"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="send_btn">
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="recieve_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="recieve_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="recieve_address_label">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="recieve_address_spinbox">
          <property name="maximum">
           <number>15000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="recieve_value_label">
          <property name="text">
           <string>Значение</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="recieve_value_lcd">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="read_btn">
        <property name="text">
         <string>Считать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
